containers/ws/install-package.sh

```diff
@@ -21,15 +21,16 @@ fi
 "$INSTALLER" install -y sed
 
 arch=`uname -p`
-rpm=$(ls /container/rpms/cockpit-ws-*$OSVER.*$arch.rpm /container/rpms/cockpit-bridge-*$OSVER.*$arch.rpm || true)
+rpm=$(ls /container/rpms/cockpit-ws-*$OSVER.*$arch.rpm /container/rpms/cockpit-bridge-*$OSVER.*$arch.rpm /container/rpms/cockpit-dashboard-*$OSVER.*$arch.rpm || true)
```

Comment:
This commit could use a rationale - i. e. on RHEL/CentOS 7, cockpit-dashboard ships ssh.